shared/actions/pinentry.js

```diff
@@ -1,21 +1,16 @@
 // @flow
 import logger from '../logger'
 import * as EngineGen from '../actions/engine-gen-gen'
-import * as ConfigGen from '../actions/config-gen'
 import * as PinentryGen from '../actions/pinentry-gen'
 import * as Saga from '../util/saga'
 import * as I from 'immutable'
 import * as RPCTypes from '../constants/types/rpc-gen'
-import engine from '../engine'
+import {getEngine} from '../engine/require'
 
-// We keep track of sessionID to response objects since this is initiated by the daemon
-type IncomingErrorCallback = (?{code?: number, desc?: string}) => void
-const sessionIDToResponse: {
```

Comment:
spoke to @maxtaco about this. this likely never worked. so lets simplify and just keep one response. in the future if we want this to be smarter its likely easier for the GUI to make a call to the daemon registering itself with an engineSaga and just getting pinenetry requests with a 'real id' and not this session id